File: clean_defaults.js
const fs = require('fs');
const path = require('path');

const manifestsDir = path.join(__dirname, 'manifests');
const files = fs.readdirSync(manifestsDir).filter(f => f.endsWith('.json') && f !== '_cards.json');

console.log(`找到 ${files.length} 个 manifest 文件\n`);

files.forEach(filename => {
  const filepath = path.join(manifestsDir, filename);
  console.log(`处理: ${filename}`);

  const data = JSON.parse(fs.readFileSync(filepath, 'utf8'));

  if (data.inputs && Array.isArray(data.inputs)) {
    let modified = false;
    data.inputs.forEach(inp => {
      if (inp.type === 'textarea' && inp.default && inp.default.trim()) {
        console.log(`  清空字段: ${inp.key} (原长度: ${inp.default.length})`);
        inp.default = '';
        modified = true;
      }
    });

    if (modified) {
      fs.writeFileSync(filepath, JSON.stringify(data, null, 1), 'utf8');
      console.log(`  ✓ 已保存\n`);
    } else {
      console.log(`  - 无需修改\n`);
    }
  } else {
    console.log(`  - 无 inputs 字段\n`);
  }
});

console.log('完成！');

File: clean_defaults.py
#!/usr/bin/env python3
"""清理所有 manifest 文件中提示词的示例文案（default 字段）"""
import json
import os
from pathlib import Path

manifests_dir = Path(__file__).parent / "manifests"

# 遍历所有 manifest 文件（除了 _cards.json）
for json_file in manifests_dir.glob("*.json"):
    if json_file.name == "_cards.json":
        continue

    print(f"处理: {json_file.name}")

    with open(json_file, 'r', encoding='utf-8') as f:
        data = json.load(f)

    # 检查 inputs 字段
    if 'inputs' in data and isinstance(data['inputs'], list):
        modified = False
        for inp in data['inputs']:
            # 如果是 textarea 类型且有 default 字段，清空它
            if inp.get('type') == 'textarea' and 'default' in inp:
                old_default = inp['default']
                inp['default'] = ""
                modified = True
                print(f"  清空字段: {inp.get('key', '?')} (原长度: {len(str(old_default))})")

        if modified:
            # 保存修改后的文件
            with open(json_file, 'w', encoding='utf-8') as f:
                json.dump(data, f, ensure_ascii=False, indent=1)
            print(f"  ✓ 已保存")
        else:
            print(f"  - 无需修改")
    else:
        print(f"  - 无 inputs 字段")

print("\n完成！")
